plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/plugin.xml

```diff
@@ -1,11 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
+
    <extension point="org.eclipse.jdt.ui.classpathContainerPage">
      <classpathContainerPage
        id="com.google.cloud.tools.eclipse.appengine.libraries.ui"
        name="App Engine Libraries"
        class="com.google.cloud.tools.eclipse.appengine.libraries.ui.AppEngineLibrariesPage">
     </classpathContainerPage>
+    <classpathContainerPage
+       id="com.google.cloud.tools.eclipse.appengine.libraries.ui"
```

Comment:
This is tricky, and I'm by no means certain I have it right. However, I believe that in the case of classpathContainerPage the id attribute is not in fact an ID, bur rather a reference to an ID defined elsewhere. Per

http://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fextension-points%2Forg_eclipse_jdt_ui_classpathContainerPage.html

```
<!ELEMENT classpathContainerPage EMPTY>
<!ATTLIST classpathContainerPage
id    IDREF #REQUIRED
name  CDATA #IMPLIED
class CDATA #IMPLIED>
```

id - identifies the classpath containers for which this page will be used. The id corresponds to the first segment of the path of a container classpath entry and matches the id referenced in org.eclipse.jdt.core.classpathContainerInitializer

So the values here both match the id attribute of the corresponding classpathContainerInitializer